test-suite/handwritten-src/cpp/test_helpers.cpp

```diff
@@ -9,6 +9,7 @@
 #include "primitive_list.hpp"
 #include "set_record.hpp"
 #include <exception>
+#include <stdexcept>
```

Comment:
MSVC suddenly complains about missing this include. Maybe this has something to do with the upgrade to C++17 ? 🤷